server.py

```diff
@@ -14,7 +14,7 @@
 
     # Configure the server object
     server.socket_host = "0.0.0.0"
-    server.socket_port = 8080
+    server.socket_port = 8888
```

Comment:
Why?
Also, if this is done other files like Dockerfile and the template in roger-cli need to be changed too.